### PR TITLE
Dockerise AYS backend

### PR DESCRIPTION
**Pull request by halilturkoglucs**
Add     Dockerfile and docker instructions to README

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM eclipse-temurin:17-jdk-alpine
```
Comment (skayikci) [inline Dockerfile:1]: suggestion: maybe we can create a folder called: docker then put every docker related files there. e.g. docker-compose files, etc.

Comment (halilturkoglucs) [reply Dockerfile:1]: I think the dockerfile should reside on the root level (same level as build result -> target folder), what do you think?

Comment (skayikci) [reply Dockerfile:1]: makes sense.

Comment (agitrubard) [reply Dockerfile:1]: Can we merge this PR? @skayikci @halilturkoglucs

Comment (halilturkoglucs) [reply Dockerfile:1]: I can merge as soon as @skayikci approves the pr

**Review by skayikci — APPROVED** [review]

**Pull request merged**